app/views/layouts/_header.html.slim

```diff
@@ -8,6 +8,8 @@ header
         li = application_disambiguation_link
       - if current_user.try :current_student?
         li = link_to 'Status Updates', [:students, :status_updates], class: params[:controller] == 'students/status_updates' ? 'active' : ''
+      - if current_user.roles.includes?('supervisor')
```

Comment:
@F3PiX we typed this code twice now, so maybe we should make a helper for this. I'm too familiar with where we should do this. wdyt teamie?